lib/charms/mongodb/v1/mongodb_provider.py

```diff
@@ -50,32 +51,27 @@
 class MongoDBProvider(Object):
     """In this class, we manage client database relations."""
 
-    def __init__(self, charm: CharmBase, substrate="k8s", relation_name: str = REL_NAME) -> None:
+    def __init__(self, charm: CharmBase, substrate="k8s", relation_name: str = "database") -> None:
```

Comment:
Is there a reason to introduce a magic string here instead of using the constant associated with it?

Reply:
good, eyes, I can do

Reply:
[addressed in this pr](https://github.com/canonical/mongodb-operator/pull/466/files)